R/render-cached-plot.R

```diff
@@ -471,6 +475,8 @@ renderCachedPlot <- function(expr,
               plotObj = drawReactiveResult,
               width = width,
               height = height,
+#jy: default to title when NULL
+            alt = alt,
```

Comment:
Also set `alt <- altWrapper()` about 40 lines up where `width` / `height` / `pixelratio` are set.

```suggestion
              alt = alt,
```